maintainers/maintainer-list.nix

```diff
@@ -1515,6 +1515,12 @@
     githubId = 977929;
     name = "Cody Allen";
   };
+  cemguresci = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add cemguresci``.

Reply:
Added.